include/SDL3/SDL_events.h

```diff
@@ -126,8 +127,10 @@ typedef enum
                                              in an event watcher, the window handle is still valid and can still be used to retrieve any userdata
                                              associated with the window. Otherwise, the handle has already been destroyed and all resources
                                              associated with it are invalid */
+    SDL_EVENT_WINDOW_PEN_ENTER,         /**< Window has gained focus of the pressure-sensitive pen with ID "data1" */
+    SDL_EVENT_WINDOW_PEN_LEAVE,         /**< Window has lost focus of the pressure-sensitive pen with ID "data1" */
     SDL_EVENT_WINDOW_FIRST = SDL_EVENT_WINDOW_SHOWN,
-    SDL_EVENT_WINDOW_LAST = SDL_EVENT_WINDOW_DESTROYED,
+    SDL_EVENT_WINDOW_LAST = SDL_EVENT_WINDOW_PEN_LEAVE,
```

Comment:
I'd ideally like them right after the `MOUSE_ENTER` events and will update the patch.  The downside is that this will shift the meaning of later event IDs.  As long as SDL3 isn't stable, that's not an issue, but if the patch doesn't make it for the first stable SDL3 release, we might need to reserve those two IDs for binary compatibility.
(Personally, I'd be delighted to have it included in the first stable SDL3 release, of course...)